src/tokens.js

```diff
@@ -332,35 +333,43 @@ const tokens = {
    * @inner
    */
   createToken(address, name, symbol, mintAmount, pin) {
+    const mintOptions = {
+      createAnotherMint: true,
+      createMelt: true,
+    };
+
+    const txData = this.createMintData(null, null, address, mintAmount, null, mintOptions);
+
+    // Set create token tx version value
+    const createTokenTxData = Object.assign(txData, {
+      version: CREATE_TOKEN_TX_VERSION,
+      name,
+      symbol,
+    });
+
+    let preparedData = null;
+    try {
+      preparedData = transaction.prepareData(createTokenTxData, pin);
+    } catch (e) {
+      const message = helpers.handlePrepareDataError(e);
+      return {success: false, message: e.message};
```

Comment:
Why do we need `const message = helpers.handlePrepareDataError(e); `? We don't use it.

Reply:
My bad, the returned message was supposed to be `message` and not `e.message`. Fixed it.